### PR TITLE
New package: python3-i3ipc-1.7.1

### PR DESCRIPTION
**Pull request by nwg-piotr**
A really useful python library for i3, works well with Sway.

### CODE REVIEW

File: srcpkgs/python3-i3ipc/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'python3-i3ipc'
+pkgname=python3-i3ipc
+version=1.7.1
+revision=1
+archs=noarch
+wrksrc="i3ipc-python-${version}"
+build_style=python3-module
+pycompile_module="python3-i3ipc"
+hostmakedepends="python3-setuptools"
+depends="python3 python3-setuptools"
```
Comment (zdtcd) [inline srcpkgs/python3-i3ipc/template]: I don't think we need `python3-setuptools` in `depends`

File: srcpkgs/python3-i3ipc/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'python3-i3ipc'
+pkgname=python3-i3ipc
+version=1.7.1
+revision=1
+archs=noarch
+wrksrc="i3ipc-python-${version}"
+build_style=python3-module
+pycompile_module="python3-i3ipc"
```
[the remark this answers is not in the document]
Comment (nwg-piotr) [reply srcpkgs/python3-i3ipc/template]: I'm not sure what you mean. Could you explain?

File: srcpkgs/python3-i3ipc/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+hostmakedepends="python3-setuptools"
+depends="python3 python3-setuptools"
+short_desc="Improved Python library to control i3wm"
+maintainer="Piotr Miller <nwg.piotr@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/acrisci/i3ipc-python"
+distfiles="${homepage}/archive/v${version}.tar.gz"
```
[the remark this answers is not in the document]
Comment (nwg-piotr) [reply srcpkgs/python3-i3ipc/template]: Why not.

File: srcpkgs/python3-i3ipc/template [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+checksum=b13f4013941e8252988e63d9b7528ba278e4af6edbce031936c5877187fc4d8c
+
+post_install() {
+	vmkdir usr/share/doc/${pkgname}/examples
+	vcopy "examples/*.py" usr/share/doc/${pkgname}/examples
```
[the remark this answers is not in the document]
Comment (nwg-piotr) [reply srcpkgs/python3-i3ipc/template]: I don't think we should modify their examples. I'd rather consider not copying them at all, if unacceptable.

File: srcpkgs/python3-i3ipc/template (hunk 1 above)
Comment (nwg-piotr) [reply srcpkgs/python3-i3ipc/template]: I wasn't sure. To be removed.

File: srcpkgs/python3-i3ipc/template (hunk 2 above)
[the remark this answers is not in the document]
Comment (zdtcd) [reply srcpkgs/python3-i3ipc/template]: `pycompile_module="i3ipc"`

[the remark this answers is not in the document]
Comment (nwg-piotr) [reply srcpkgs/python3-i3ipc/template]: Thanks!

File: srcpkgs/python3-i3ipc/template (hunk 3 above)
[the remark this answers is not in the document]
Comment (zdtcd) [reply srcpkgs/python3-i3ipc/template]: It's more like a convention.

[the remark this answers is not in the document]
Comment (nwg-piotr) [reply srcpkgs/python3-i3ipc/template]: One way or another - I wish they released more often. v1.7.1 misses bug fixes merged in May. :/

[the remark this answers is not in the document]
Comment (zdtcd) [reply srcpkgs/python3-i3ipc/template]: Both releases in PyPI and GitHub are 1.7.1.

[the remark this answers is not in the document]
Comment (nwg-piotr) [reply srcpkgs/python3-i3ipc/template]: That's true, alas. :/ On my other machine I use the `-git` version. Well, those are just small fixes, nothing critical.

[the remark this answers is not in the document]
Comment (nwg-piotr) [reply srcpkgs/python3-i3ipc/template]: The PYPI archive does not contain the LICENSE file, and xlint displays a warning:

```
$ xlint srcpkgs/python3-i3ipc/template
srcpkgs/python3-i3ipc/template:13: license 'BSD', but no use of vlicense
```
Could I give up on PYPI and use the GitHub asset?

File: srcpkgs/python3-i3ipc/template (hunk 1 above)
Comment (nwg-piotr) [reply srcpkgs/python3-i3ipc/template]: Done.

File: srcpkgs/python3-i3ipc/template (hunk 4 above)
[the remark this answers is not in the document]
Comment (nwg-piotr) [reply srcpkgs/python3-i3ipc/template]: Done.

File: srcpkgs/python3-i3ipc/template (hunk 3 above)
[the remark this answers is not in the document]
Comment (nwg-piotr) [reply srcpkgs/python3-i3ipc/template]: Got back to the GitHub release.

File: srcpkgs/python3-i3ipc/template (hunk 2 above)
[the remark this answers is not in the document]
Comment (nwg-piotr) [reply srcpkgs/python3-i3ipc/template]: Done.

**Comment by nwg-piotr** [comment]
Will submit again as soon as a fixed version > 1.7.1 appears.

**Pull request closed without merging**